### ISSUE TITLE
Too big values in ops/s error column

### ISSUE THREAD

**Issue by plokhotnyuk** [japgolly/scalajs-benchmark#88]
![image](https://user-images.githubusercontent.com/890289/82136230-a5af1b00-980b-11ea-9259-58b5c0b68596.png)

**Comment by japgolly** [comment]
LGTM, what's the problem?

**Comment by plokhotnyuk** [comment]
I expect that error values for ops/s to be less than ops/s values if an error in micros/op is low.

As example for the last benchmark it should be (+/-7952):
```
scala> Math.max(Math.abs(1000000 / (4.537) - 1000000 / (4.537 - 0.158)), 1000000 / (4.537) - 1000000 / (4.537 + 0.158))
res0: Double = 7952.677341810573
```

Please see how JMH Visualizer represents throughput errors:
![image](https://user-images.githubusercontent.com/890289/82137059-c2038580-9814-11ea-9771-cc4aed13439e.png)

**Comment by japgolly** [comment]
Oh I see, I thought you meant rendering, not the values themselves. I thought it was a CSS problem or something. Yeah those numbers look ridiculous. I'll take a look after I finish with batch mode.

**Comment by japgolly** [comment]
Hey @plokhotnyuk , can you attach the JMH JSON file from scalajs-benchmark for this case?

**Comment by plokhotnyuk** [comment]
Here it is but from the Chrome browser:
```
[
  {
    "benchmark" : "PrimitivesReading.uPickle",
    "mode" : "avgt",
    "threads" : 1,
    "forks" : 1,
    "jdkVersion" : "1.8",
    "vmName" : "Scala.JS",
    "vmVersion" : "1.1.0",
    "userAgent" : "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.77 Safari/537.36",
    "warmupIterations" : 2,
    "warmupTime" : "1 s",
    "warmupBatchSize" : 1,
    "measurementIterations" : 9,
    "measurementTime" : "1 s",
    "measurementBatchSize" : 1,
    "primaryMetric" : {
      "score" : 11.606,
      "scoreError" : 0.28800000000000003,
      "scoreConfidence" : [
        11.317,
        11.894
      ],
      "scoreUnit" : "us/op",
      "rawData" : [
        [
          11.457,
          11.367,
          11.558,
          11.608,
          11.618,
          11.485999999999999,
          11.779,
          11.754000000000001,
          11.84
        ]
      ]
    },
    "secondaryMetrics" : {
      
    }
  },
  {
    "benchmark" : "PrimitivesReading.jsoniterScala",
    "mode" : "avgt",
    "threads" : 1,
    "forks" : 1,
    "jdkVersion" : "1.8",
    "vmName" : "Scala.JS",
    "vmVersion" : "1.1.0",
    "userAgent" : "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.77 Safari/537.36",
    "warmupIterations" : 2,
    "warmupTime" : "1 s",
    "warmupBatchSize" : 1,
    "measurementIterations" : 9,
    "measurementTime" : "1 s",
    "measurementBatchSize" : 1,
    "primaryMetric" : {
      "score" : 1.315,
      "scoreError" : 0.028,
      "scoreConfidence" : [
        1.286,
        1.3439999999999999
      ],
      "scoreUnit" : "us/op",
      "rawData" : [
        [
          1.282,
          1.459,
          1.286,
          1.286,
          1.307,
          1.287,
          1.303,
          1.342,
          1.3010000000000002
        ]
      ]
    },
    "secondaryMetrics" : {
      
    }
  },
  {
    "benchmark" : "PrimitivesReading.borer",
    "mode" : "avgt",
    "threads" : 1,
    "forks" : 1,
    "jdkVersion" : "1.8",
    "vmName" : "Scala.JS",
    "vmVersion" : "1.1.0",
    "userAgent" : "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.77 Safari/537.36",
    "warmupIterations" : 2,
    "warmupTime" : "1 s",
    "warmupBatchSize" : 1,
    "measurementIterations" : 9,
    "measurementTime" : "1 s",
    "measurementBatchSize" : 1,
    "primaryMetric" : {
      "score" : 2.778,
      "scoreError" : 0.058,
      "scoreConfidence" : [
        2.719,
        2.836
      ],
      "scoreUnit" : "us/op",
      "rawData" : [
        [
          2.984,
          2.684,
          2.705,
          2.7560000000000002,
          2.797,
          2.742,
          2.781,
          2.787,
          2.785
        ]
      ]
    },
    "secondaryMetrics" : {
      
    }
  },
  {
    "benchmark" : "PrimitivesReading.circe",
    "mode" : "avgt",
    "threads" : 1,
    "forks" : 1,
    "jdkVersion" : "1.8",
    "vmName" : "Scala.JS",
    "vmVersion" : "1.1.0",
    "userAgent" : "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.77 Safari/537.36",
    "warmupIterations" : 2,
    "warmupTime" : "1 s",
    "warmupBatchSize" : 1,
    "measurementIterations" : 9,
    "measurementTime" : "1 s",
    "measurementBatchSize" : 1,
    "primaryMetric" : {
      "score" : 15.402000000000001,
      "scoreError" : 0.281,
      "scoreConfidence" : [
        15.121,
        15.683
      ],
      "scoreUnit" : "us/op",
      "rawData" : [
        [
          15.423,
          15.323,
          15.395000000000001,
          15.296000000000001,
          15.234,
          15.391,
          15.610000000000001,
          15.291,
          15.665
        ]
      ]
    },
    "secondaryMetrics" : {
      
    }
  },
  {
    "benchmark" : "PrimitivesReading.avSystemGenCodec",
    "mode" : "avgt",
    "threads" : 1,
    "forks" : 1,
    "jdkVersion" : "1.8",
    "vmName" : "Scala.JS",
    "vmVersion" : "1.1.0",
    "userAgent" : "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.77 Safari/537.36",
    "warmupIterations" : 2,
    "warmupTime" : "1 s",
    "warmupBatchSize" : 1,
    "measurementIterations" : 9,
    "measurementTime" : "1 s",
    "measurementBatchSize" : 1,
    "primaryMetric" : {
      "score" : 7.352,
      "scoreError" : 0.098,
      "scoreConfidence" : [
        7.2540000000000004,
        7.45
      ],
      "scoreUnit" : "us/op",
      "rawData" : [
        [
          7.339,
          7.3660000000000005,
          7.348999999999999,
          7.243,
          7.383,
          7.590999999999999,
          7.2940000000000005,
          7.284,
          7.329
        ]
      ]
    },
    "secondaryMetrics" : {
      
    }
  }
]
```

**Comment by japgolly** [comment]
Thanks for reporting this, @plokhotnyuk . Fixed in #92. New release soon :D
